ssn/rdf/sosa.ttl

```diff
@@ -219,21 +260,23 @@ sosa:isHostedBy
   rdf:type owl:ObjectProperty ;
   schema:domainIncludes sosa:Actuator ;
   schema:domainIncludes sosa:Sensor ;
+  schema:domainIncludes sosa:Sampler ;
   schema:rangeIncludes sosa:Platform ;
-  rdfs:comment "Relation between a Sensor or Actuator and the Platform that it is mounted on or hosted by."@en ;
+  rdfs:comment "Relation between a Sensor or Actuator or Sampler and the Platform that it is mounted on or hosted by."@en ;
```

Comment:
"Relation between a Sensor, or Actuator, or Sampler" to stay in line with the style used in SOSA